recipes/vibration.py

```diff
@@ -50,8 +51,11 @@ class VibrationsJob(MultiJob):
     """
     _result_type = VibrationsResults
 
-    def __init__(self, molecule, settings, jobType=ADFJob, get_gradients='get_gradients', reorder='inputOrder', aseVibOpt={}):
-        MultiJob.__init__(self)
+    def __init__(self, molecule, settings, name='plams.vib', jobType=ADFJob, get_gradients='get_gradients', reorder='inputOrder', aseVibOpt={}):
```

Comment:
```suggestion
    def __init__(self, molecule, settings, jobType=ADFJob, get_gradients='get_gradients', reorder='inputOrder', aseVibOpt={}, name='plams.vib'):
```
Probably a bit safer this way, in case someone was passing `jobType` as a positional argument.